friendly_id.gemspec

```
@@ -25,10 +25,10 @@ Gem::Specification.new do |s|
   s.add_development_dependency "ffaker"
   s.add_development_dependency "simplecov"
 
-  s.description = <<~EOM
-    FriendlyId is the "Swiss Army bulldozer" of slugging and permalink plugins for
-    Active Record. It lets you create pretty URLs and work with human-friendly
-    strings as if they were numeric ids.
+  s.description = <<-EOM
+FriendlyId is the "Swiss Army bulldozer" of slugging and permalink plugins for
+Active Record. It lets you create pretty URLs and work with human-friendly
+strings as if they were numeric ids.
   EOM
```

```suggestion
  s.description = "FriendlyId is the \"Swiss Army bulldozer\" of slugging and" \
    "permalink plugins for Active Record. It lets you create pretty URLs and " \
    "work with human-friendly strings as if they were numeric ids."
```

What about this version, so that standard doesn't change it back, @Bastes ?

---

Agreed :)

---

And done.